src/v/security/audit/audit_log_manager.cc

```diff
@@ -829,4 +831,22 @@ audit_log_manager::should_enqueue_audit_event(event_type type) const {
     return std::nullopt;
 }
 
+std::optional<audit_log_manager::audit_event_passthrough>
+audit_log_manager::should_enqueue_audit_event(
+  kafka::api_key key, const model::topic& t) const {
+    const auto& excluded_topics = _audit_excluded_topics();
+    if (!excluded_topics.empty()) {
+        if (std::any_of(
+              excluded_topics.begin(),
+              excluded_topics.end(),
+              [&t](const ss::sstring& topic_name) {
+                  return t() == topic_name;
+              })) {
+            return std::make_optional(audit_event_passthrough::yes);
+        }
+    }
+
+    return should_enqueue_audit_event(key);
```

Comment:
> I guess its debatable if we want that behavior though. Maybe we should modify things so that anything that would return passthrough::no gets evaluated last.

Yeah this is probably more appropriate